s2wav/s2let_math.py

```diff
@@ -0,0 +1,83 @@
+"""Compute basic math tiling. """
+import numpy as np
+
+
+def f_s2dw(k, B):
+    """Tiling function for S2DW wavelets"""
```

Comment:
Doc strings needs to follow google style and be more extensive (should define arguments and returns).